ccd-definition/FixedLists.json

```diff
@@ -122,109 +122,109 @@
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "LivingSituationList",
-    "ListElementCode": "Other",
+    "ListElementCode": "OTHER",
     "ListElement": "Other"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "FathersResponsibilityList",
-    "ListElementCode": "Yes",
+    "ListElementCode": "YES",
     "ListElement": "Yes"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "FathersResponsibilityList",
-    "ListElementCode": "No",
+    "ListElementCode": "NO",
     "ListElement": "No"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "FathersResponsibilityList",
-    "ListElementCode": "Don't know",
+    "ListElementCode": "DON'T_KNOW",
     "ListElement": "Don't know"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "PastOrFutureHarmSelect",
-    "ListElementCode": "Past harm",
+    "ListElementCode": "PAST_HARM",
     "ListElement": "Past harm"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "PastOrFutureHarmSelect",
-    "ListElementCode": "Future risk of harm",
+    "ListElementCode": "FUTURE_RISK_OF_HARM",
     "ListElement": "Future risk of harm"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "ProceedingStatus",
-    "ListElementCode": "Ongoing",
+    "ListElementCode": "ONGOING",
     "ListElement": "Ongoing"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "ProceedingStatus",
-    "ListElementCode": "Previous",
+    "ListElementCode": "PREVIOUS",
     "ListElement": "Previous"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "AllocationProposalList",
-    "ListElementCode": "Lay justices",
+    "ListElementCode": "LAY_JUSTICES",
     "ListElement": "Lay justices"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "AllocationProposalList",
-    "ListElementCode": "District judge",
+    "ListElementCode": "DISTRICT_JUDGE",
     "ListElement": "District judge"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "AllocationProposalList",
-    "ListElementCode": "Circuit judge",
+    "ListElementCode": "CIRCUIT_JUDGE",
     "ListElement": "Circuit judge"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "AllocationProposalList",
-    "ListElementCode": "Section 9 circuit judge",
+    "ListElementCode": "SECTION_9_CIRCUIT_JUDGE",
     "ListElement": "Section 9 circuit judge"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "AllocationProposalList",
-    "ListElementCode": "High Court judge",
+    "ListElementCode": "HIGH_COURT_JUDGE",
     "ListElement": "High Court judge"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "GenderList",
-    "ListElementCode": "Male",
+    "ListElementCode": "MALE",
     "ListElement": "Male"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "GenderList",
-    "ListElementCode": "Female",
+    "ListElementCode": "FEMALE",
     "ListElement": "Female"
   },
   {
     "LiveFrom": "01/01/2017",
     "ID": "GenderList",
-    "ListElementCode": "They identify in another way",
+    "ListElementCode": "THEY_IDENTIFY_IN_ANOTHER_WAY",
```

Comment:
is this AND  
{
    "LiveFrom": "01/01/2017",
    "ID": "ChildGenderList",
    "ListElementCode": "THEY_IDENTIFY_IN_ANOTHER_WAY",
    "ListElement": "They identify in another way"
  },

Sorry to add more work, but we could refactor one out